Composer/packages/adaptive-form/src/components/SchemaField.tsx

```diff
@@ -83,6 +95,11 @@ const SchemaField: React.FC<FieldProps> = (props) => {
     schema,
     uiOptions,
     value: deserializedValue,
+    onFocus: () => setFieldFocused(true),
+    onBlur: () => {
+      setFieldFocused(false);
+      rest.onBlur && rest.onBlur(rest.id);
```

Comment:
This looks strange to me.  You spread rest in the fieldProps, but then are assuming it has onBlur here.  What if rest doesn't have onBlur?  It would be nice to see if rest conformed to a HTML element type that has onBlur.

Reply:
I am not 100% sure I understand what you mean here. I deconstructed onBlur on the props to leave _rest_ out of this. Was that what you had in mind?